velox/functions/sparksql/DecimalArithmetic.cpp

```diff
@@ -458,65 +467,102 @@ struct DecimalDivideFunction {
       uint8_t aScale,
       uint8_t bPrecision,
       uint8_t bScale) {
```

Comment:
Comment suggestion:

>   // When allowing precision loss, computes the result precision and scale
  // following Hive's formulas. If result is representable with long decimal,
  // the result scale is the maximum of 'aScale' and 'bScale'. If not, reduces
  // result scale and caps the result precision at 38. When denying precision
  // loss, calculates the number of whole digits and fraction digits. If the
  // total number of digits exceed 38, we reduce both the number of fraction
  // digits and whole digits to fit within this limit.

Reply:
This is not right for divide, it is `std::max(6, aScale + bPrecision + 1)`.
This function is a private function of class in anonymous namespace, and the computation is precise by code,  do we really need to clarify the computation in comment? And for allow precision loss, the detailed comments exists in `DecimalUtil::adjustPrecisionScale(precision, scale)`, so I will comment it shortly.
> If result is representable with long decimal,
// the result scale is the maximum of 'aScale'  and 'bScale'